src/templates/amfcfg.conf.j2

```diff
@@ -0,0 +1,164 @@
+configuration:
+  amfDBName: {{ amf_database_name }}
+  amfName: AMF
+  debugProfilePort: 5001
+  enableDBStore: false
+  enableSctpLb: false
+  kafkaInfo:
```

Comment:
Since this section is not used, I wonder if we can simply remove it. Hard to tell I suppose without having the whole core running.